packages/components/src/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import { Col, FormControl, Row } from 'react-bootstrap';
+
+import { List } from 'immutable';
+
+import { SectionHeading } from '../SectionHeading';
+
+import { DomainFieldLabel } from '../DomainFieldLabel';
+import { LoadingSpinner, Principal, SelectInput } from '../../..';
+
+import { IssuesListDefModel } from './models';
+import {UserGroup} from "../../permissions/models";
+import {
+    ISSUES_LIST_DEF_SORT_DIRECTION_TIP,
+    ISSUES_LIST_DEF_SINGULAR_PLURAL_TIP,
+    ISSUES_LIST_GROUP_ASSIGN_TIP, ISSUES_LIST_USER_ASSIGN_TIP
+} from "./constants";
+import { getCoreGroups, getCoreUsersInGroups } from "../../permissions/actions";
+import produce from "immer";
+
+interface IssuesListDefBasicPropertiesInputsProps {
+    model: IssuesListDefModel;
+    onInputChange?: (any) => void;
+    onSelect?: (selected: String, name?: String) => void;
```

Comment:
@labkey-bpatel I didn't notice the casing issue (String -> string) on the type here:
```suggestion
    onSelect?: (selected: string, name?: string) => void;
```

Reply:
whoops! now fixed! Good catch!